hack/istio/playbooks/files/bookinfo.yaml

```diff
@@ -18,9 +18,9 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: details
+  name: details-service${LABEL}
```

Comment:
does this work? I believe the bookinfo code hardcodes the names of the services - so you renaming the service "details-service-${LABEL}" is going to break that code - I bet if you run this, you'll get 100% errors going to all these services because you renamed them.

This is why we need to append ${LABEL} to the end of the app labels. If it was just a matter of renaming the service  names to have "-service" prefixes at the end, then we would have no need for ${LABEL} at all - since the service name and app label values would all be different. But we can't rename the services - they have to remain "details" and "reviews" and "ratings" because the bookinfo application code literally hardcodes those values. Thus, we need to keep the service names the same and add ${LABEL} to the end of only the app label values.